app/views/verify/sessions/new.html.slim

```diff
@@ -21,18 +21,22 @@ h1.h3.my0 = t('idv.titles.session.basic')
         = f.input :state, collection: us_states_territories,
           label: t('idv.form.state'), required: true
       .sm-col.sm-col-4.px1
-        = f.input :zipcode, label: t('idv.form.zipcode'), required: true,
+        / using :tel for mobile numeric keypad
+        = f.input :zipcode, as: :tel,
+          label: t('idv.form.zipcode'), required: true,
           pattern: '(\d{5}([\-]\d{4})?)',
           input_html: { class: 'zipcode', value: idv_profile_form.zipcode }
   .mb4
-    .mt0.mb3.pb-tiny.border-bottom.border-teal = t('profile.index.dob')
```

Comment:
I thought the same thing at first, but it's still used here: https://github.com/18F/identity-idp/blob/master/app/views/profile/index.html.slim#L36, just hard to spot because the relative naming style.